src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -514,10 +514,10 @@
     short-summary: Enable vertical pod autoscaler for cluster.
   - name: --nodepool-allowed-host-ports
     type: string
-    short-summary: Expose host ports on the node pool. When specified, format should be a comma-separated list of ranges with protocol, eg. 80/TCP,443/TCP,4000-5000/TCP.
+    short-summary: Expose host ports on the node pool. When specified, format should be a space-separated list of ranges with protocol, eg. 80/TCP,443/TCP,4000-5000/TCP.
```

Comment:
If this should be a space-separated list, the example should be `80/TCP 443/TCP 4000-5000/TCP` instead of `80/TCP,443/TCP,4000-5000/TCP`.

Reply:
And this is a behavior change compared to aks-preview, is this accepted by PM?